src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -31,6 +31,10 @@
     <form action="/login" method="POST">
       <label for="username">Username: </label>
       <input type="text" name="username" id="username">
+	<br/>
```

Comment:
Let's keep the indentation consistent here (2 spaces for nested elements)

Reply:
By that I mean it looks like you're using tabs here instead of spaces (more of a nit than anything, doesn't matter too much)

Reply:
Oh okay, I might have just tabbed the code out of habit. Thank you. Is the code okay to be merged right now or should I make more corrections?

Reply:
LGTM!